### PR TITLE
soft skills section OK

### LINKED ISSUE #136: add manfred2md soft skills section

### PR DESCRIPTION
**Pull request by JuanBerrocal**
closed #136

### CODE REVIEW

**Comment by JuanBerrocal** [comment]
closed #145

**Comment by JuanBerrocal** [comment]
closed #136

File: packages/manfred2md/src/engine/md-parts/soft-skills-section/soft-skills-section.ejs
Comment (juanpms2) [inline packages/manfred2md/src/engine/md-parts/soft-skills-section/soft-skills-section.ejs:1]: Replace by: 

```
---

### SOFT SKILLS

<%_ if (softSkillsCollection && softSkillsCollection !== 'undefined') { -%>
  <%- softSkillsCollection.map(element => element.skill.name).join(' / ') %>
<%_ } -%>
```

Comment (JuanBerrocal) [reply packages/manfred2md/src/engine/md-parts/soft-skills-section/soft-skills-section.ejs:1]: OK. Changes made.

**Pull request merged**